src/posix/sdl/i_system.cpp

```diff
@@ -157,6 +158,29 @@ bool gameisdead;
 void Mac_I_FatalError(const char* errortext);
 #endif
 
+#ifdef __linux__
+void Linux_I_FatalError(const char* errortext)
+{
+	// Close window or exit fullscreen and release mouse capture
+	SDL_Quit();
+
+	const char *str;
+	if((str=getenv("XDG_CURRENT_DESKTOP")) && strcmp(str, "KDE") == 0)
+	{
+		FString cmd;
+		cmd << "kdialog --title \"" GAMESIG " ";
+		cmd << GetVersionString() << ": No IWAD found\" ";
+		cmd << "--msgbox \"" << errortext << "\"";
+		popen(cmd, "r");
+	} else if (I_GtkAvailable())
+	{
+		Linux_I_FatalError_Gtk(errortext);
+	} else {
+		SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, (std::string("") + GAMESIG + ": No IWAD found").c_str(), errortext, NULL);
```

Comment:
I don't think you should use `std::string` here. You should do the same thing with `FString` in the `KDE` branch above.

Reply:
Done